### PR TITLE
Support for relatedImages

### PR DESCRIPTION
**Pull request by sabre1041**
Support for disconnected environments

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -273,8 +301,13 @@ jobs:
 
       - name: create and copy bundle to community operators
         shell: bash
+        env:
+          QUAY_REGISTRY: "${{ env.QUAY_REGISTRY }}"
+          REPOSITORY_NAME: "${{ env.REPOSITORY_NAME }}"
+          OPERATOR_IMAGE_TAG: "v${{ env.VERSION }}"
         run: |
           make bundle IMG=quay.io/${QUAY_REGISTRY}:v${VERSION} VERSION=${VERSION} DEFAULT_CHANNEL=alpha
+          ./hack/attach_image_digests.sh
```
Comment (raffaelespazzoli) [inline .github/workflows/push.yml:310]: why do you have to do the hack with all of those replaces? Would it be simpler to just get the sha from the built image and build the bundle with that sha?

Comment (sabre1041) [reply .github/workflows/push.yml:310]: 1. Has to be done after bundle is built (certain components in Deployments dont yet exist)
2. When building for OperatorHub, there will be no image locally anyways as it is built in a different job

Comment (sabre1041) [reply .github/workflows/push.yml:310]: @raffaelespazzoli bump

Comment (raffaelespazzoli) [reply .github/workflows/push.yml:310]: mmm not convinced, but if you want to got his way that's fine.

**Pull request merged**